schemas/input/csl-data.json

```
@@ -468,8 +473,19 @@
         }
       ]
     },
+    "edtf-datatype": {
+      "title": "EDTF datatype pattern",
+      "description": "CSL input supports EDTF, validated against this regular expression.",
+      "type": "string",
+      "pattern": "[0-9,-,%,~,X,?,..,\/]"
+    },
     "date-variable": {
+      "title": "Date content model.",
+      "description": "The CSL input model supports two different date representations: an EDTF string, and a more structured equivalent.",
```

```suggestion
      "description": "The CSL input model supports two different date representations: a preferred EDTF string, and a more structured equivalent.",
```

Possible modification, to prioritize EDTF.

---

I would suggest:
`…an EDTF string (preferred)…`